x-pack/plugins/ml/public/application/util/time_series_explorer_service.ts

```diff
@@ -25,22 +25,39 @@ import { timeBucketsServiceFactory } from './time_buckets_service';
 import type { TimeRangeBounds } from './time_buckets';
 import type { Job } from '../../../common/types/anomaly_detection_jobs';
 import type { TimeBucketsInterval } from './time_buckets';
-import type {
-  ChartDataPoint,
-  FocusData,
-  Interval,
-} from '../timeseriesexplorer/timeseriesexplorer_utils/get_focus_data';
 import type { CriteriaField } from '../services/results_service';
 import {
   MAX_SCHEDULED_EVENTS,
   TIME_FIELD_NAME,
 } from '../timeseriesexplorer/timeseriesexplorer_constants';
 import type { MlApiServices } from '../services/ml_api_service';
 import { mlResultsServiceProvider, type MlResultsService } from '../services/results_service';
-import { forecastServiceProvider } from '../services/forecast_service_provider';
+import { forecastServiceFactory } from '../services/forecast_service';
 import { timeSeriesSearchServiceFactory } from '../timeseriesexplorer/timeseriesexplorer_utils/time_series_search_service';
 import { useMlKibana } from '../contexts/kibana';
 
+export interface Interval {
+  asMilliseconds: () => number;
+  expression: string;
+}
+
+export interface ChartDataPoint {
```

Comment:
Nit, these don't need to be exported anymore as they are only used inside this file.

Reply:
Updated in 70a4278a495651f8acb2d17f57cf79570ac71c26